src/vl-map-search.js

```diff
@@ -123,19 +125,45 @@ export class VlMapSearch extends vlElement(HTMLElement) {
       this.__searchEventListenerRegistered = true;
       this._selectElement.addEventListener('search', (event) => {
         if (event && event.detail && event.detail.value) {
-          fetch(this.searchUrl + event.detail.value).then((response) => {
-            return response.json();
-          }).then((data) => {
-            if (data && data.SuggestionResult) {
-              const resultaten = data.SuggestionResult.map((resultaat) => {
-                return {
-                  value: resultaat,
-                  label: resultaat,
-                };
-              });
-              this._selectElement.choices = resultaten;
-            }
-          });
+          const lambertCoordinaat = LambertCoordinaat.of(event.detail.value);
+
+          if (lambertCoordinaat === undefined) {
+            fetch(this.searchUrl + event.detail.value).then((response) => {
+              return response.json();
+            }).then((data) => {
+              if (data && data.SuggestionResult) {
+                const resultaten = data.SuggestionResult.map((resultaat) => {
+                  return {
+                    value: resultaat,
+                    label: resultaat,
+                  };
+                });
+                this._selectElement.choices = resultaten;
+              }
+            });
+          } else {
+            fetch(this._findLocationsByLambertCoordinaatUrl(lambertCoordinaat)).then((response) => {
+              return response.json();
```

Comment:
Denk dat dit strikt genomen zelfs niet nodig was, om nog eens te zoeken naar adressen voor dat coordinaat. 
Maar nu het er dan toch is mag het er ook wel in blijven zitten vermoed ik @kevcraey ?

Reply:
je hebt gelijk, moet het mislezen hebben en zo in mijn hoofd blijven hangen zijn. Maar idd vind dit op zich niet zo slecht die extra suggesties.